services/helper/mentors.js

```diff
@@ -20,17 +27,49 @@ module.exports = class MentorsHelper {
         })
     }
 
-    static reports(userId) {
-        return new Promise(async (resolve,reject) => {
-            try {
+    static async reports(userId, filterType) {
+        let filterStartDate;
+        let filterEndDate;
+        let totalSessionCreated;
+        let totalsessionHosted;
+        let filters;
+        try {
+            if (filterType === 'MONTHLY') {
+                [filterStartDate, filterEndDate] = utils.getCurrentMonthRange();
+            } else if (filterType === 'WEEKLY') {
+                [filterStartDate, filterEndDate] = utils.getCurrentWeekRange();
+            } else if (filterType === 'QUARTERLY') {
+                [filterStartDate, filterEndDate] = utils.getCurrentQuarterRange();
+            }
 
-                /**
-                 * Your business logic here
-                 */
+            /* totalSessionCreated */ 
+            filters = {
+                createdAt: {
+                    $gte: filterStartDate.toISOString(),
+                    $lte: filterEndDate.toISOString()
+                },
+                userId,
+                deleted: false
+            };
 
-            } catch(error) {
-                return reject(error);
-            }
-        })
+            totalSessionCreated = await sessionsData.countSessions(filters);
+
+            /* totalsessionHosted */ 
+            filters = {
+                startDateUtc: {
+                    $gte: filterStartDate.toISOString(),
+                    $lte: filterEndDate.toISOString()
+                },
+                userId,
+                status: 'completed',
```

Comment:
@aman-g202  What will happen if the session is live ? Mentor has started the sessions which means it is hosted right ?